builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -147,24 +147,24 @@ func resourceAwsS3Bucket() *schema.Resource {
 			},
 
 			"versioning": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
Gonna put my ignorance on display here for a moment: would this break configs/require a migration?